example-service/Cargo.toml

```diff
@@ -13,7 +13,7 @@ readme = "../README.md"
 description = "An example server built on tarpc."
 
 [dependencies]
-tarpc-json-transport = { version = "0.1", path = "../json-transport" }
+tarpc-bincode-transport = { version = "0.7", path = "../bincode-transport" }
```

Comment:
Any particular reason to use bincode over json? The nice thing about json for examples is that the protocol is human-readable, so easier to debug/play around with when starting out.